src/Microsoft.AspNetCore.HttpOverrides/ForwardedHeadersMiddleware.cs

```diff
@@ -185,7 +185,7 @@ public void ApplyForwarders(HttpContext context)
                     }
                     else if (_options.RequireHeaderSymmetry)
                     {
-                        _logger.LogDebug(4, $"Failed to parse forwarded host: {set.Host}");
+                        _logger.LogWarning(4, $"Forwarded host is not present, this is required by {nameof(_options.RequireHeaderSymmetry)}");
```

Comment:
"Incorrect number of x-forwarded-proto header values, see RequireHeaderSymmetry."